pkgs/tools/package-management/nix/default.nix

```diff
@@ -87,15 +87,5 @@ in lib.makeExtensible (self: {
 
   stable = self.nix_2_8;
 
-  # remember to backport updates to the stable branch!
-  unstable = lib.lowPrio (common rec {
-    version = "2.8";
-    suffix = "pre20220530_${lib.substring 0 7 src.rev}";
-    src = fetchFromGitHub {
-      owner = "NixOS";
-      repo = "nix";
-      rev = "af23d38019a47e5bb4cd6585a1678b37c957130c";
-      sha256 = "sha256-RH77Y4IhbTofNYlLQSGKLL0fJAG9iHSwRNvMEZ4M0VQ=";
-    };
-  });
+  unstable = self.stable;
```

Comment:
i'd prefer not to remove it but if it is wanted then this should be turned in to a throw alias that explains why it has been removed

Reply:
The title says "remove" but to me it's more like increasing version of `nixUnstable` to 2.8.